charts/istio-alerts/templates/service-prometheusrule.yaml

```diff
@@ -32,4 +32,27 @@ spec:
         {{- end }}
     {{- end }}
     {{- end }}
+
+     # Adding latency alarm to trigger when request latency is above > threshold
+    {{ with .Values.serviceRules.highRequestLatency }}
+    - alert: HighRequestLatency
+      annotations:
+        summary: Processes experience latency above {{ .threshold }}.
+        runbook_url: {{ $values.defaults.runbookUrl }} #HighRequestLatency
+        description: >-
+          {{`{{ $value | humanizePercentage }} latency is above
+          namespace {{ $labels.namespace }} for container {{ $labels.container
+          }} in pod {{ $labels.pod }}.`}}
+      expr: |-
+        sum(irate(istio_request_duration_milliseconds_bucket{reporter=~"destination",destination_service=~"featurestore-featurestore-api.featurestore.svc.cluster.local"}[1m])) by (le)) / 1000) by (container, pod, namespace)
```

Comment:
you'll want to replace the `destination_service` filter with a filter on namespace to match the 5xx alarm (i.e. `destination_service_namespace="{{ $release.Namespace }}`)